### PR TITLE
Update setup script

### PR DESCRIPTION
**Pull request by naiteeks**
~ moved deployment manifest file to .clouddrive\edge-deployment\
~ removed ACR references and replaced with mcr location
~ added logging options in deployment manifest
~ added APPDATA_FOLDER_ON_DEVICE to env file
~ create binding for APPDATA_FOLDER_ON_DEVICE in lvaEdge module
~ fixed validation errors for the deployment file
~ added a print of the location of the deployment file

### CODE REVIEW

File: edge/setup/setup.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ BLUE='\033[1;34m'
 NC='\033[0m' # No Color
 
 # script configuration
-BASE_URL='https://raw.githubusercontent.com/bennage/live-video-analytics/setup-script/edge/setup' # location of remote files used by the script
+BASE_URL='https://raw.githubusercontent.com/naiteeks/live-video-analytics/naiteeks-updates/edge/setup' # location of remote files used by the script
```
Comment (bennage) [inline edge/setup/setup.sh:21]: Ultimately, this should be pointing to files hosted in this repo and not in a fork.
Like:
`https://raw.githubusercontent.com/Azure/live-video-analytics/master/edge/setup/`

Comment (naiteeks) [reply edge/setup/setup.sh:21]: Yes that’s right. I think once we get the final build as well, we will need to update the script as well. So then we can fix that as well.

File: edge/setup/setup.sh [hunk 2]
```diff
@@ -301,10 +302,11 @@ echo "IOTHUB_CONNECTION_STRING=$IOTHUB_CONNECTION_STRING" >> $ENV_FILE
 echo "AAD_TENANT_ID=$AAD_TENANT_ID" >> $ENV_FILE
 echo "AAD_SERVICE_PRINCIPAL_ID=$AAD_SERVICE_PRINCIPAL_ID" >> $ENV_FILE
 echo "AAD_SERVICE_PRINCIPAL_SECRET=$AAD_SERVICE_PRINCIPAL_SECRET" >> $ENV_FILE
-echo "INPUT_VIDEO_FOLDER_ON_DEVICE=\"$INPUT_VIDEO_FOLDER_ON_DEVICE\"" >> $ENV_FILE
-echo "OUTPUT_VIDEO_FOLDER_ON_DEVICE=\"$OUTPUT_VIDEO_FOLDER_ON_DEVICE\"" >> $ENV_FILE
-echo "CONTAINER_REGISTRY_USERNAME_myacr=\"$CONTAINER_REGISTRY_USERNAME\"" >> $ENV_FILE
-echo "CONTAINER_REGISTRY_PASSWORD_myacr=\"$CONTAINER_REGISTRY_PASSWORD\"" >> $ENV_FILE
+echo "INPUT_VIDEO_FOLDER_ON_DEVICE=\"/home/lvaadmin/samples/input\"" >> $ENV_FILE
+echo "OUTPUT_VIDEO_FOLDER_ON_DEVICE=\"/home/lvaadmin/samples/output\"" >> $ENV_FILE
```
Comment (bennage) [inline edge/setup/setup.sh:306]: Hard-coding the path could be problematic.
If we are hard-coding, we should remove the `INPUT_VIDEO_FOLDER_ON_DEVICE` variables from the top of the script.

File: edge/setup/setup.sh (hunk 1 above)
Comment (bennage) [reply edge/setup/setup.sh:21]: 👍 
I also highly recommend that we use a tag for the URL and not the master branch.
This is to make sure we don't break the script with updates to master.

File: edge/setup/setup.sh (hunk 2 above)
Comment (naiteeks) [reply edge/setup/setup.sh:304]: fixed it in the new commit

**Comment by bennage** [comment]
lgtm

File: MediaGraph/topologies/evr-motion-files/topology.json [hunk 3]
```diff
@@ -1,93 +1,93 @@
...
+                "description": "file sink output name",
+                "default": "filesinkOutput"
+            }
+        ],
+        "sources": [
+            {
+                "@type": "#Microsoft.Media.MediaGraphRtspSource",
+                "name": "rtspSource",
+                "endpoint": {
+                    "@type": "#Microsoft.Media.MediaGraphClearEndpoint",
```
Comment (gadamilan) [inline MediaGraph/topologies/evr-motion-files/topology.json:42]: Are you going back to clear endpoint?
Why is this file part of this PR?

**Pull request closed without merging**